### DISCUSSION TITLE
Change the behavior of no fail

### DISCUSSION THREAD

**Discussion by Awesome-Aasim** [ppy/osu#30418] [Ideas] [answered]
I want to see if we can change the behavior of No Fail in Lazer. Essentially, if a person has no fail enabled and their HP bar does not drop to zero during the game, the score should be treated the same as if a person did not have No Fail enabled (i.e. multipled by 1x rather than 0.5x). This is more a quality of life thing because No Fail should kind of be the default except for the most competitive players.

**Comment by bdach** [comment, marked as answer]
We do not want to encourage widespread usage of NF in any capacity.

See also:
https://github.com/ppy/osu/discussions/28706
https://github.com/ppy/osu/discussions/21865
https://github.com/ppy/osu/discussions/18424
https://github.com/ppy/osu/discussions/16779
https://github.com/ppy/osu/discussions/14114
https://github.com/ppy/osu/issues/9844
